decimer.py

```diff
@@ -0,0 +1,126 @@
+"""Decimer test. This code will have to be moved somewhere else before merging to master"""
+
+# https://arxiv.org/pdf/<id>
+
+import requests
+import pdf2image  # Debian: apt install poppler-utils / Alpine: apk add poppler
+from PIL import Image
+import numpy as np
+import logging
+import decimer_segmentation  # Debian: apt install libgl1
+import DECIMER
+import rdkit.Chem
+from skfp.fingerprints import AtomPairFingerprint
+from aslite.db import EmbeddingsDB, get_papers_db
+from aslite import config
+from pymilvus import MilvusClient
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(name)s %(levelname)s %(asctime)s %(message)s",
+    datefmt="%Y-%m-%d %H:%M:%S",
+)
+
+
+def get_pdf_url(id_: str, source: str = "arxiv") -> str:
+    if source == "arxiv":
+        return f"https://arxiv.org/pdf/{id_}"
+    elif source == "chemrxiv":
+        r = requests.get(
+            f"https://chemrxiv.org/engage/chemrxiv/public-api/v1/items/{id_}"
+        )
+        r.raise_for_status()
+        return r.json()["asset"]["original"]["url"]
+    else:
+        raise ValueError("'source' must be equal to 'arxiv' or 'chemrxiv'.")
+
+
+def download_pdf_as_images(url: str) -> list[np.ndarray]:
+    # Warning: the whole paper is downloaded to RAM (TODO: check if it's really a problem at scale).
```

Comment:
ale nie jest tak że w danym momencie potrzebujecie mieć w ramie 1 papier, więc to nie szkodzi, bo pojedynczy paper zajmuje mało?